spec/sidekiq_unique_jobs/lock_config_spec.rb

```diff
@@ -76,7 +76,7 @@
     it { is_expected.to eq(:log) }
 
     context "when on_conflict is a hash" do
-      let(:on_conflict) { { client: :replace, server: :reschedule } }
+      let(:on_conflict) { { "client" => :replace, "server" => :reschedule } }
```

Comment:
Should match what happens on the client - only keys are stringified

Reply:
Actually no. In the previous PR I make sure that these are tested properly. This is also not the tests for the file you changed.

In fact this change makes it even less _real_ since most likely the on conflict will be a symbol and my tests here would no longer catch if they are a test.

Reply:
OK, I've removed the changes to this test. Thanks for the explanation!